loops-and-arrays.js

```diff
@@ -11,6 +11,12 @@ const names = ["Henk", "Piet", "Fred", "Joop"];
 // console.log(names) geeft: ["Henk", "Piet", "Fred", "Joop"]
 // Na jouw script zie je de aangepaste waardes:
 // console.log(names) geeft: ["Henkje", "Pietje", "Fredje", "Joopje"]
+for(let i = 0; i < names.length; i++){
```

Comment:
Eventueel zou je de for-lus kunnen vervangen door een map-functie voor meer leesbaarheid en modernere JavaScript-praktijken. Bijvoorbeeld:
const names = names.map(name => name + "je");